README.md

```diff
@@ -1,2 +1,33 @@
-# Ner4Opt
-Ner4Opt: Named-Entity Recognition for Optimization
+# Named Entity Recognition for Optimization (Ner4Opt) Library
+
+Given an optimization problem in natural language, the library extracts **six named entities**:
+
+- **CONST_DIR** (constraint direction)
```

Comment:
can you use ** CONST_DIR**: Constraint direction and drop the paranthesis thing..